README.md

```diff
@@ -1,2 +1,249 @@
+
+Furthermore, the shepherd creates an issue (or issues) that tracks the
+implementation of the RFC in the appropriate repository (or repositories) and
+inserts links at the top section of the RFC document. The shepherd may also
+delegate this task to the RFC author or another person that would be responsible
+for implementing the RFC.
+
+#### The role of the shepherd
+[The role of the shepherd]: #role-of-the-shepherd
+
+The shepherd is a trusted individual who understands the community well and
+knows the major stakeholders. Typically, this will be a
+[maintainer](https://github.com/orgs/lf-lang/teams/maintainers). During the RFC
+process, the shepherd acts as a moderator and is responsible for identifying and
+notifying the relevant stakeholders. The main task of the shepherd is to steer
+the discussions towards a conclusion. The shepherd also decides when to start
+the FCP period to bring the RFC to the attention of the maintainers who make the
+final decision.
+
+## Implementation of active RFCs
+[Implementation of active RFCs]: #implementation-of-actions-rfcs
+
+Once an RFC becomes active, anyone may submit an implementation of the feature
+as a pull request to the relevant repo. Being active, however, does not
+guarantee that an implementation of the feature will ultimately be merged; it
+does mean that, in principle, all the major stakeholders have agreed to the
+feature and are amenable to merging it.
+
+The nature of software development is that new insights are gained during
+implementation and requirements change frequently. Thus, no design document can
+perfectly reflect the end product. Active RFCs are not a specification and you
+may deviate from the initial design where needed (this is even encouraged!).
+Please document those deviations in the implementation PR. Active RFCs may also
+be modified in a follow-up pull-request to incorporate new insights. However, if
+a more significant change to the design is required, a new RFC should be created
+with a note added to the original RFC.
+
+A core idea of the RFC process is to think about the implications of a new
+feature and its design early on, and to receive feedback quickly and frequently.
+This helps to identify potential concerns and major roadblocks before investing
+in a concrete implementation. However, it might be helpful to support arguments
+for or against an RFC with a prototype implementation. For some high-priority
+features, it might also be required to work on the implementation alongside the
+RFC. However, if you decide to invest in an implementation before the RFC
+becomes active, the existence of this implementation and the work that you put
+in do not present arguments against concerns raised by the RFC reviewers.
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make any semantic or syntactic
```

Comment:
I think we might be a bit too strict here. I can suggest a rewrite of this paragraph.